packages/jsdoc-plugins/tests/observable-event-provider/addmissingeventdocletsforobservables.js

```diff
@@ -0,0 +1,251 @@
+/**
+ * @license Copyright (c) 2016-2017, CKSource - Frederico Knabben. All rights reserved.
+ * Licensed under the terms of the MIT License (see LICENSE.md).
+ */
+
+'use strict';
+
+const { expect } = require( 'chai' );
+const addMissingEventDocletsForObservables = require( '../../lib/observable-event-provider/addmissingeventdocletsforobservables' );
+
+describe( 'jsdoc-plugins/observable-event-provider', () => {
+	describe( 'addMissingEventDocletsForObservables()', () => {
+		it( 'should provide new event doclet for the observable property', () => {
+			const inputDoclets = [ {
+				comment: '...',
+				meta: {
+					range: [
+						1861,
+						2061
+					],
+					filename: 'command.js',
+					lineno: 56,
+					path: '/Users/maciejbukowski/workspace/ckeditor5/packages/ckeditor5-core/src',
```

Comment:
You should not use real paths in tests. What in case I will change something in this file. Shall I correct the paths because my workspace is different?

Let's use `/workspace/ckeditor5/packages/ckeditor5-core/src` instead.

Reply:
Hmm, right, didn't catch this. Tests don't use them at all, so they'll work everywhere. I just wanted to add a real case example.